### PR TITLE
Pn 53 sitter gallery

### PR DESCRIPTION
**Pull request by wowawawoa**
What I have done:
1. pet sitter can add photos through pet sitter profile page
image
2. pet sitter can resize image when upload 
image
3. pet sitter can remove image
image
4. pet sitter can reorder images 
image
5. pet sitter's images can be viewed on pet sitter page
image
6. pet owner can view pet sitter's images with full screen
image
7. add responsive on pet sitter page
image

### CODE REVIEW

File: src/components/PetSitterProfile/ProfileGallery/Popup/Popup.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+          title: "Image upload failed",
+          status: "error",
+          duration: 3000,
+          isClosable: true,
+          containerStyle: { fontSize: "20px", maxWidth: "400px", padding: "10px" },
+        });
+      }
+    }
+  }, [isUploadSuccess, isUploadError]);
+
```
Comment (jube33717f) [inline src/components/PetSitterProfile/ProfileGallery/Popup/Popup.tsx:108]: usually, we define variables first, then call hooks, then define functions

File: src/components/PetSitterProfile/ProfileGallery/ProfileImages/ProfileImages.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+  const petOwner = useSelector((state: any) => state.petOwner);
+  const { data: imagesData, refetch: refetchImages } = useUserGetOwnImagesQuery(petOwner._id);
+
+  const [updateOrder, { isSuccess: isUpdateOrderSuccess }] = useUpdateImagesOrderMutation(
+    petOwner._id
+  );
+
+  useEffect(() => {
+    if (imagesData !== undefined) {
```
Comment (jube33717f) [inline src/components/PetSitterProfile/ProfileGallery/ProfileImages/ProfileImages.tsx:34]: =`if(imagesData)`

**Review by jube33717f — CHANGES_REQUESTED** [review]
looks good, just some minor things can improve

**Comment by wowawawoa** [comment]
update according to the review

**Review by ShawnWang86b — DISMISSED** [review]

**Comment by wowawawoa** [comment]
solved all conversation, updated with dev

**Review by jube33717f — APPROVED** [review]

**Pull request merged**